main/src/library/flix/actor/Actor.flix

```diff
@@ -14,7 +14,7 @@
 /// @param s the type of state.
 /// @param e the type of the error.
 ///
-pub enum ActorResult[s, e] {
+pub enum ActorResult[e] {
```

Comment:
I would like to rename ActorResult. I am currently leaning towards `Outcome`. Do you have a better suggestion?

Reply:
Either `Outcome` or `ActorOutcome` would be fine